lib/controllers.js

```diff
@@ -1,57 +1,59 @@
 'use strict';
 
-var groups = require.main.require('./src/groups');
+const groups = require.main.require('./src/groups');
 
-var Controllers = {};
+const Controllers = {};
 
 Controllers.renderAdminPage = async (req, res) => {
 	const groupData = await groups.getGroupsFromSet('groups:visible:createtime', 0, -1);
 	res.render('admin/plugins/session-sharing', { groups: groupData });
 };
 
-Controllers.retrieveUser = function (req, res) {
+Controllers.retrieveUser = async (req, res) {
```

Comment:
Missing `=>`

Reply:
Obviously I left them to make sure you were reading it and has nothing to do with bad copy paste from my version of the plugin 😞 Rechecked and I had left them out in a number of functions so have updated the code in below commits